akka-persistence/src/main/scala/akka/persistence/serialization/MessageSerializer.scala

```diff
@@ -30,6 +30,8 @@ class MessageSerializer(val system: ExtendedActorSystem) extends BaseSerializer
   val PersistentImplClass = classOf[PersistentImpl]
   val AtLeastOnceDeliverySnapshotClass = classOf[AtLeastOnceDeliverySnap]
 
+  private lazy val serialization = SerializationExtension(system)
```

Comment:
initializaton order of extension and serializer, I think the SerializationExtension creates this instance